recipe/meta.yaml

```
@@ -22,23 +22,23 @@ source:
     - fix-customize-build.patch
 
 build:
-  number: 1
-  skip: true   # [py<39 or s390x]
+  number: 2
+  skip: true  # [py<310 or s390x]
```

https://github.com/cgohlke/imagecodecs/blob/v2024.9.22/setup.py#L753
```suggestion
  skip: true  # [py<39]
```